Formula/mu.rb

```diff
@@ -48,6 +48,8 @@ class Mu < Formula
 
   fails_with gcc: "5"
 
+  conflicts_with "mu-repo", because: "both install `mu` binaries"
+
```

Comment:
Do I need to add this `conflicts_with` block in this commit/pull request or a separate one?

Reply:
Let's add the conflicts after this PR is merged.